admin/restore.php

```diff
@@ -278,7 +279,7 @@ function restore($filnavn,$backup_encode,$backup_dbtype){
   print "<!-- Saldi-kommentar for at skjule uddata til siden \n"; # Indsat da svar fra pg_dump kan resultere i besked genereres
```

Comment:
Just a note - this is found several places.
A much better solution is to redirect stdout to /dev/null - and stderr to errorlog